### PR TITLE
Check for correct pool name on ms cluster

### PR DESCRIPTION
**Pull request by fbalak**
Fixes https://github.com/red-hat-storage/ocs-ci/issues/5945

Signed-off-by: Filip Balak <fbalak@redhat.com>

### CODE REVIEW

**Review by jilju — DISMISSED** [review]

**Comment by fbalak** [comment]
Verified: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/13035
pool found

File: ocs_ci/ocs/fiojob.py [hunk 1]
```diff
@@ -312,7 +312,13 @@ def get_pool_name(fixture_name):
     Return ceph pool name based on fixture name suffix.
     """
     if fixture_name.endswith("rbd"):
-        ceph_pool_name = "ocs-storagecluster-cephblockpool"
+        if config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS:
```
Comment (ebondare) [inline ocs_ci/ocs/fiojob.py:315]: As far as I understand, if you run this code on provider, there will be no cephblockpool with provider cluster's clusterId.

Comment (fbalak) [reply ocs_ci/ocs/fiojob.py]: Yes. The code was updated to work only with consumer but this is only a partial fix.

Comment (ebondare) [reply ocs_ci/ocs/fiojob.py]: I think we could just return None if the cluster is Provider.

**Comment by yitzhak12** [comment]
I ran a PR validation job using MS provider and MS consumer when the MS consumer is the primary cluster: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-odf-multicluster/1971/console. The pr validation job passed successfully.

**Review by yitzhak12 — APPROVED** [review]

**Review by ebondare — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**